docs/api/core.md

```diff
@@ -5,79 +5,89 @@
 
 .. autoclass:: AECEnv
 
-    .. py:attribute:: agents
+```
+
+## Attributes
+
+
+```{eval-rst}
+
+.. autoattribute:: AECEnv.agents
 
-        A list of the names of all current agents, typically integers. These may be changed as an environment progresses (i.e. agents can be added or removed).
+    A list of the names of all current agents, typically integers. These may be changed as an environment progresses (i.e. agents can be added or removed).
 
-        :type: list[AgentID]
+    :type: List[AgentID]
 
-    .. py:attribute:: num_agents
+.. autoattribute:: AECEnv.num_agents
```

Comment:
Why do types sometime appear and sometimes not?

Reply:
The code is missing typehints.